app/uk/gov/hmrc/customs/inventorylinking/export/controllers/actionbuilders/AuthAction.scala

```diff
@@ -179,8 +179,8 @@ class AuthAction @Inject()(
     }
     for {
       customsEnrolment <- maybeCustomsEnrolment
-      eoriIdentifier <- customsEnrolment.getIdentifier("EORINumber")
-    } yield Eori(eoriIdentifier.value)
+      eori <- customsEnrolment.getIdentifier("EORINumber")
```

Comment:
Why this name change from `eoriIdentifier` to `eori`?

Reply:
as discussed decided to remove identifier so as not to confuse it with with submitter identifier or eori identifier